src/System.Net.Http/tests/UnitTests/System.Net.Http.Unit.Tests.csproj

```diff
@@ -8,10 +8,10 @@
     <NoWarn Condition="'$(TargetGroup)' == 'netfx'">$(NoWarn);0436</NoWarn>
     <DefineConstants Condition="'$(TargetGroup)' == 'netfx'">$(DefineConstants);NET46</DefineConstants>
   </PropertyGroup>
-  <!-- Help VS understand available configurations -->
-  <!-- Do not reference these assemblies from the TargetingPack since we are building part of the source code for tests. -->
```

Comment:
This second comment can probably remain (it's referring to the `TargetingPackExclusions` list below this block).